Dockerfile

```diff
@@ -3,18 +3,35 @@ FROM elifesciences/peerscout_client:${commit} AS client
 FROM elifesciences/python:cacd250ec201feab491c0738f261561b5360997b
```

Comment:
In some projects this tag has become a build argument so that it can be set in `.env`, making it more visible and easier to (automatically) update

Reply:
I haven't touched that. Maybe leave that for when we want to update it. I believe Dependabot would actually check the `FROM` statement (if we were using release tags). Not sure if it detects them in `.env`. I am also not yet sure about the value of our own python base image. It doesn't make clear what Python version we are using (we could use better tags) or the purpose (better name?).